msbuild/Xamarin.Localization.MSBuild/MSBStrings.resx

```diff
@@ -1,1726 +1,1503 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Looks like there are a lot of unrelated changes in this file (was it reformatted?)

Reply:
Yeah, I opened it with VS and I think it reformatted it completely :/

Reply:
Done